### DISCUSSION TITLE
How to Add Latest Distillation Techniques such as DIST or KD++ to this framework

### DISCUSSION THREAD

**Discussion by nighting0le01** [yoshitomo-matsubara/torchdistill#377] [Q&A] [answered]
How to Add Latest Distillation Techniques such as DIST or KD++ to this framework

**Comment by yoshitomo-matsubara** [comment, marked as answer]
Hi @arpitsahni04 

If you explain how those methods should work precisely
- what teacher and student models and datasets are used
- how their loss functions are designed
- which layers of the teacher and student models are used to compute the losses

then, I can explain or give it a try
